ext/cl_khr_command_buffer.asciidoc

```diff
@@ -378,6 +380,11 @@ cl_int clGetCommandBufferInfoKHR(
     size_t* param_value_size_ret);
 ----
 
+The following SVM entry points are included only from version 2.0, adding selected commands
```

Comment:
Is "from version 2.0" the right description?  Or, should this be "from version 0.9.4"?

Reply:
Clarified. It now mentions both the min OpenCL version as well as the minimum ext version.